handlers/CaseStudies/components/CaseStudiesContent/index.js

```diff
@@ -13,13 +13,10 @@ class CaseStudiesContent extends Component {
     return (
       <div className="CaseStudiesContent">
         {this.props.caseStudies.map((f, imageUrl) => (
-          (imageUrl = lookup(f.image, 'fields.file.url')),
-          <Link key={f.slug} to="article" params={{slug: f.slug}} className="CaseStudiesContent-item">
+          (imageUrl = lookup(f.caseStudyGridImage, 'fields.file.url')),
+          <Link key={f.slug} to="article" params={{slug: f.slug}} className="CaseStudiesContent-item" style={{backgroundImage: 'url(' + imageUrl + ')'}}>
             <span className="CaseStudiesContent-title">{f.title}</span>
             <span className="CaseStudiesContent-description">{f.summary}</span>
-            {imageUrl && (
-              <img src={imageUrl + '?w=400'} className="CaseStudiesContent-image" />
```

Comment:
just to note: you can append `?w={width}` to the image to get it back relevant to the viewport. This could be a useful future optimization.